docker-compose.yml

```diff
@@ -1,17 +1,17 @@
-# version: "3.7"
+version: "3.7"
 
-# services:
-#   web:
-#     build: .
-#     environment:
-#       NODE_ENV:
-#       MONGODB_URI: mongodb://mongo:27017/nodegoat
-#     command: sh -c "until nc -z -w 2 mongo 27017 && echo 'mongo is ready for connections' && node artifacts/db-reset.js && npm start; do sleep 2; done"
-#     ports:
-#       - "4000:4000"
+services:
+  web:
+    build: .
+    environment:
+      NODE_ENV:
+      MONGODB_URI: mongodb://mongo:27017/nodegoat
+    command: sh -c "until nc -z -w 2 mongo 27017 && echo 'mongo is ready for connections' && node artifacts/db-reset.js && npm start; do sleep 2; done"
+    ports:
+      - "4000:4000"
 
-#   mongo:
-#     image: mongo:4.4
-#     user: mongodb
-#     expose:
-#       - 27017
+  mongo:
+    image: mongo:4.4
```

Comment:
#### Automated PR Comment From coverity
<!-- de498386d539c55f80b384d7e49d877e -->
<!-- Comment managed by Git Commenter, do not modify! -->
### Coverity Issue - Least privilege violation
_Low CWE-284_
The Docker container is not configured to drop all default capabilities and then add only required one. Thus the container may be granted excessive privileges which violate the least privilege principle.
### How to fix
Remove all the default capabilities from the list of capabilities by setting the `cap_drop` value to `ALL` and then add only specific capabilities needed by a container.